[WIP] export label positions

olivierdalang opened the pull request:
WIP for the implementation of exporting label position

- [x] basic logic works (at least for haltung_text, not checked for abwasserbauwerk_text yet)
- [ ] ili2pg for some reason does not export haltung_text (some flags on the command line ? or another reason like topic/model/basket issue ?)
- [ ] make scale customizable
- [ ] generating label placement for the whole map at small scale is very slow on the demo dataset, see if we can improve that

3nids commented on qgepqwat2ili/gui/__init__.py:
```diff
@@ -198,12 +200,47 @@ def action_do_export():
+        structures_lyr = structures_lyrs[0]
+        reaches_lyr = reaches_lyrs[0]
+
+        QApplication.processEvents()
```
why do you need this?

olivierdalang replied:
Slightly improves UX (otherwise not properly redrawing when the progress dialog changes size). Quite minor though, but since it was already done like this for other updates, I think it's fine to leave it.

3nids commented on qgepqwat2ili/processing_algs/extractlabels_interlis.py:
```diff
@@ -0,0 +1,190 @@
+    QgsWkbTypes,
+)
+
+from ....processing_provider.qgep_algorithm import QgepAlgorithm
```
since this is meant to be used in the plugin, shouldn't we use absolute imports with qgepplugin.qgepqwat2ili ?

olivierdalang commented:
Updated the UI and added check for pre-3.26.2 versions

![image](https://user-images.githubusercontent.com/1894106/186131850-1b28eb75-9c7d-495e-ba38-dd94af8d5133.png)
![image](https://user-images.githubusercontent.com/1894106/186131913-ac4997e3-035d-462f-83e1-a1e9e8ce4379.png)

olivierdalang commented:
@sjib Ok was able to reproduce issue described in https://github.com/QGEP/QGEP/issues/552#issuecomment-1272220658. Ability to add geom without explicit SRID only exists since Postgis 3.1 (didn't expect such a difference in minor versions).

I have pushed a fix ready to test (still same PR: https://github.com/QGEP/qgepplugin/pull/69)
I also added Postgis 3.0 to the CI matrix.

olivierdalang commented:
Todo:
- [ ] store selection in the settings (the first time, none are selected)
